src/context/AppProvider/AppContext.tsx

```diff
@@ -4,6 +4,10 @@ import {
   toRootDerivationPath,
   utxoAccountParams,
 } from '@shapeshiftoss/chain-adapters'
+import {
+  UTXOBaseAdapter,
+  UTXOChainTypes,
+} from '@shapeshiftoss/chain-adapters/dist/utxo/UTXOBaseAdapter'
```

Comment:
diiiiiist

Reply:
I'll ask the dev that changed removed all the dist occurrences to do the same here